cmd/oras/attach.go

```diff
@@ -152,11 +152,24 @@ func runAttach(opts attachOptions) error {
 				}
 				return content.Successors(ctx, fetcher, node)
 			}
+		} else if root.MediaType == ocispec.MediaTypeImageManifest {
+			graphCopyOptions.FindSuccessors = func(ctx context.Context, fetcher content.Fetcher, node ocispec.Descriptor) ([]ocispec.Descriptor, error) {
+				if content.Equal(node, root) {
+					// skip subject
+					return append(descs, ocispec.Descriptor{
+						// scratch config blob with content of `{}` (size of 2)
+						MediaType: "application/vnd.oci.example+json",
+						Digest:    `sha256:44136fa355b3678a1146ad16f7e8649e94fb4fc21fe77e8310c060f61caaff8a`,
+						Size:      2,
+						Data:      []byte(`{}`)}), nil
+				}
+				return content.Successors(ctx, fetcher, node)
+			}
```

Comment:
The point of line 147 - 168 is to skip the `subject` node. That is, remove the `subject` node from the successors if the current node is a root node.